actions/changelog-generate/action.yaml

```diff
@@ -63,7 +80,7 @@ runs:
       id: git-cliff
       with:
         config: tmp_cliff.toml
-        args: "--tag ${{ inputs.tag }} --verbose"
+        args: "--tag ${{ inputs.tag }}"
```

Comment:
Shouldn't this include `--unreleased` as well, so we can generate the changelog for the version that we are about to release?

Reply:
this is available either via the range attribute or the latest changelog only options